lekcja10/główny.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+<header>
+   <div class="login" style="text-align: end; padding: 15px; margin: -5px; background-color: antiquewhite;">
```

Comment:
Dlaczego style inline? Efektywniej bedzie je zdefiniowac w zewnetrznym arkuszu.

PS. Sprawdz caly kod, tych stylei inline jest znacznie wiecej.

Reply:
Czy to poprawiles?